### ISSUE TITLE
Allow Attributes in HTML Form

### ISSUE THREAD

**Issue by dandrzejewski** [knadh/listmonk#2201]
**Describe the solution you'd like**
I would like the ability to add custom inputs to the HTML forms to capture additional information about users that will be stored as attributes.

For example, if I added: 

`<p><input type="text" name="topping" placeholder="Favorite Pizza Topping" /></p>`

to an HTML form, I'd like it to store the attribute `topping` for that subscriber.

For that matter, it might be nice to be able to include these attributes on the built-in form at `/subscription/form`.

**Comment by srikary12** [comment]
I'm willing to work on this :)

**Comment by knadh** [comment]
@dandrzejewski please refer to this: https://github.com/knadh/listmonk/issues/105

**Comment by dandrzejewski** [comment]
> @dandrzejewski please refer to this: #105

Yeah.... I was kind of hoping for a no-code solution to this, but I understand.
